### ISSUE TITLE
Greasing version negotiation

### ISSUE THREAD

**Issue by mirjak** [quicwg/base-drafts#1668]
I know this was discussed and concluded early on but I would like to bring this up again as probably the plan for versioning quic quickly has changed since.

The transport draft currently says:
"Versions that follow the pattern 0x?a?a?a?a are reserved for use in
   forcing version negotiation to be exercised.  That is, any version
   number where the low four bits of all octets is 1010 (in binary).  A
   client or server MAY advertise support for any of these reserved
   versions."

My question: Are people actually implemening and using this? Because forcing version negotiation means additional latency. I would assume that publishing a next version quickly and actually using version negotiation as indented is the better greasing.

So basically this is a quick check if we still want to have that text. I assume we can quickly agree and close this issue again!

**Comment by MikeBishop** [comment]
Yes, actually *using* multiple versions is the best way, but this has value as well.  I'd expect to see this in servers mostly, padding out VN packets with reserved versions.  Clients that aren't latency sensitive might do an initial probe with a junk payload and a reserved version occasionally, but I don't expect browsers will very often.

**Comment by mnot** [comment]
Discussed in NYC; there is still substantial support for this in the WG. Closing with no action.
